internal/gomod/stdlib.go

```diff
@@ -0,0 +1,232 @@
+package gomod
+
+// IsStandardlibPackge determines whether a package is in the standard library
+// or not. At this point, it checks whether the package name is one of those
+// that is found from running "go list std" in the latest released go version.
+func IsStandardlibPackge(pkg string) bool {
+	_, ok := standardLibraryMap[pkg]
+	return ok
+}
+
+var contained interface{}
+
+// This list is calculated from "go list std".
+//
+// You can get a formattable golang snippet by:
+//
+//    $ echo "var standardLibraryMap = map[string]interface{}{" \
+//       && go list std | awk '{ print "\""$0"\": contained,"}' \
+//       && echo "}"
+//
+var standardLibraryMap = map[string]interface{}{
```

Comment:
Can we define this in a file on its own, and add a bash script to the repo that dumps that content to the file?

Reply:
Also, nice strategy. Thanks @tjdevries and @Strum355 for working through these tricky edge cases!

Reply:
ty ty :) added script to generate completely